web/src/pages.rs

```diff
@@ -146,3 +151,77 @@ pub async fn my_articles(
     })
     .into_response())
 }
+
+#[axum::debug_handler(state = AppState)]
+pub async fn my_categories(
+    t: Template,
+    State(state): State<AppState>,
+    user: User,
+) -> WebResult<Response> {
+    info!("Fetching categories for user_id: {}", user.user_id);
+
+    let categories = sqlx::query!(
+        "SELECT DISTINCT c.* 
+     FROM category c
+     JOIN categorymarkdown cm ON c.category_id = cm.category_id
```

Comment:
I think we can simplify the join syntax just a touch here! And this is one reason we wanted the column names to 'match' on both tables!

We should be able to do something like this instead of doing the `ON stuff!

```suggestion
     JOIN categorymarkdown cm USING (category_id)
```